code/game/machinery/_machines_base/machine_construction/pipe.dm

```diff
@@ -9,7 +9,6 @@
 /decl/machine_construction/pipe/proc/deconstruct_transition(obj/item/I, mob/user, obj/machinery/machine)
 	if(isWrench(I))
```

Comment:
Missing tool interaction.
```suggestion
	if(isWrench(I) && I.do_tool_interaction(TOOL_WRENCH, user, machine, 0, "unfastening", "unfastening"))
```
Maybe even remove `isWrench(I)` since `I.do_tool_interaction()` checks for the wrench archetype.